README.md

```diff
@@ -0,0 +1,204 @@
+# Gimbal Airship Adapter - Cordova Plugin
+
+The Gimbal Airship Adapter Cordova Plugin integrates Gimbal Place Event triggers with Airship Custom Events in Cordova apps.
+
+## Resources
+
+- [Gimbal Developer Guide](https://gimbal.com/doc/android/v4/devguide.html)
+- [Gimbal Manager Portal](https://manager.gimbal.com)
+- [Airship Getting Started guide](https://docs.airship.com/platform/android/getting-started/)
+- [Airship and Gimbal Integration guide](https://docs.airship.com/partners/gimbal/)
+
+## Installation
+
+### Prerequisites
+
+In [Gimbal Manager > Apps](https://manager.gimbal.com/apps) create an app for each iOS and Android.
+(For the purposes of differentiating between them in analytics, it helps to make the names and bundle/package IDs unique.)
+You can find your apps' API keys in the app detail view -- you will need these later.
+
+### Add the plugin
+
+To add the plugin to your app:
+
+```shell
+    cordova plugin add cordova-plugin-gimbal-airship-adapter
+```
+
+## Initialization
```

Comment:
Should we have a blurb here about inserting the Airship key and secret as well?

Reply:
good point, I think something about requiring the urbanairship-cordova plugin and its configuration is in order.